webserver/sign.py

```diff
@@ -0,0 +1,18 @@
+import socket
+import hashlib
+import struct
+
+data = open('data', 'rb').read()
+hashed_data = hashlib.sha3_256(data).digest()
+
+with socket.socket() as sock:
+    sock.connect(('localhost', 1337))
+    
+    sock.sendall(hashed_data)
+    print('Data sent.')
+
+    date = sock.recv(10)
+    sig = sock.recv(64)
```

Comment:
I dont like these consts, consider to put them in config.py or something, but it's up to you